core/templates/pages/admin-page/platform-parameters-tab/admin-platform-parameters-tab.component.ts

```diff
@@ -143,12 +151,47 @@ export class AdminPlatformParametersTabComponent implements OnInit {
     return {type: dataType};
   }
 
-  addNewRuleToTop(param: PlatformParameter): void {
-    param.rules.unshift(cloneDeep(this.getdefaultNewRule(param)));
+  getReadonlyFilterValues(rule: PlatformParameterRule): string {
+    let resultantString: string = '';
+    let filters: PlatformParameterFilter[] = rule.filters;
+    for (let filterIdx = 0; filterIdx < filters.length; filterIdx++) {
+      let filterName: string = (
```

Comment:
use `const` if you do not reassign variables

Reply:
I see even `filterIdx` and `filterName` are not being reassigned again. Change them to const too. Ideally, always use const by default unless you have to reassign the variable later.

Reply:
Ah!, I missed the `fileName` variable. I have updated that. For `filterIdx`, I cannot really convert it to const as it gets increased in the for loop and trying to convert it to const gives error.
I have updated all other relevant places.
Thanks